packages/edit-site/src/components/sidebar-navigation-screen-template/home-template-details.js

```diff
@@ -0,0 +1,223 @@
+import { useMemo, useState, useEffect } from '@wordpress/element';
+import { decodeEntities } from '@wordpress/html-entities';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SidebarNavigationScreenDetailsPanel,
+	SidebarNavigationScreenDetailsPanelRow,
+} from '../sidebar-navigation-screen-details-panel';
+import { unlock } from '../../lock-unlock';
+import { store as editSiteStore } from '../../store';
+import { useLink } from '../routes/link';
+import SidebarNavigationItem from '../sidebar-navigation-item';
+
+const EMPTY_OBJECT = {};
+
+function TemplateAreaButton( { postId, icon, title } ) {
+	const icons = {
+		header,
+		footer,
+	};
+	const linkInfo = useLink( {
+		postType: 'wp_template_part',
+		postId,
+	} );
+
+	return (
+		<SidebarNavigationItem
+			className="edit-site-sidebar-navigation-screen-template__template-area-button"
+			{ ...linkInfo }
+			icon={ icons[ icon ] ?? layout }
+			withChevron
+		>
+			<Truncate
+				limit={ 20 }
+				ellipsizeMode="tail"
+				numberOfLines={ 1 }
+				className="edit-site-sidebar-navigation-screen-template__template-area-label-text"
+			>
+				{ decodeEntities( title ) }
+			</Truncate>
+		</SidebarNavigationItem>
+	);
+}
+
+export default function HomeTemplateDetails() {
+	const navigator = useNavigator();
+	const {
+		params: { postType, postId },
+	} = navigator;
+	const { editEntityRecord } = useDispatch( coreStore );
+
+	const {
+		allowCommentsOnNewPosts,
+		templatePartAreas,
+		postsPerPage,
+		postsPageTitle,
+		postsPageId,
+		templateRecord,
+	} = useSelect(
+		( select ) => {
+			const { getEntityRecord } = select( coreStore );
+			const siteSettings = getEntityRecord( 'root', 'site' );
+			const { getSettings } = unlock( select( editSiteStore ) );
+			const siteEditorSettings = getSettings();
+			const _templateRecord =
+				select( coreStore ).getEditedEntityRecord(
+					'postType',
+					postType,
+					postId
+				) || EMPTY_OBJECT;
+			const _postsPageRecord = siteSettings?.page_for_posts
+				? select( coreStore ).getEntityRecord(
+						'postType',
+						'page',
+						siteSettings?.page_for_posts
+				  )
+				: EMPTY_OBJECT;
+
+			return {
+				templateRecord: _templateRecord,
+				allowCommentsOnNewPosts:
+					siteSettings?.default_comment_status === 'open',
+				postsPageTitle: _postsPageRecord?.title?.rendered,
+				postsPageId: _postsPageRecord?.id,
+				postsPerPage: siteSettings?.posts_per_page,
+				templatePartAreas: siteEditorSettings?.defaultTemplatePartAreas,
+			};
+		},
+		[ postType, postId ]
+	);
+
+	const [ commentsOnNewPostsValue, setCommentsOnNewPostsValue ] =
+		useState( '' );
+	const [ postsCountValue, setPostsCountValue ] = useState( 1 );
+	const [ postsPageTitleValue, setPostsPageTitleValue ] = useState( '' );
+
+	useEffect( () => {
+		setCommentsOnNewPostsValue( allowCommentsOnNewPosts );
+		setPostsPageTitleValue( postsPageTitle );
+		setPostsCountValue( postsPerPage );
+	}, [ postsPageTitle, allowCommentsOnNewPosts, postsPerPage ] );
```

Comment:
This syncs the server-retrieved values to local state, yeah? Only if you're making further changes anyway, would it be worth adding a comment just before this useEffect? It took me a second to work out what was happening.

Reply:
It would be worth it. Added to follow up notes. Thank you!